src/main/java/zd/zero/waifu/motivator/plugin/integrations/ErrorReporter.kt

```diff
@@ -136,4 +136,41 @@ class ErrorReporter : ErrorReportSubmitter() {
         if (edition != null) appName += " ($edition)"
         return Pair(appInfo, appName)
     }
+
+    private fun getMinifiedConfig(): String {
+        val keyMapper = fun(s: String): String {
+            val capitals = s.codePoints()
+                .filter { Character.isUpperCase(it) }
+                .mapToObj { Character.toChars(it).joinToString() }
+                .collect(Collectors.joining())
+
+            return s[0].plus(capitals.toLowerCase())
+        }
+        val valueMapper = fun(s: String): String = when {
+            "true".equals(s, ignoreCase = true) -> "1"
+            "false".equals(s, ignoreCase = true) -> "0"
+            else -> s
+        }
+        val minifiedConfig = JsonObject()
```

Comment:
> Now, I want to change this into something, if we can just avoid the JSON format at all

This doesn't need to be a JSON object. I was just lazy and let GSON do a pretty `toString` for me. If you had other ideas, go for it.

I had an idea where you can just use the `index` of the config as the key and the value is the minified version of it.
You can also omit the `user-id` as that shows up in other places (saves some space)

ex: `0:1;1:1;2:0.....10:0,130;......N:(someValue)`

Where something like this `<index of the config>:<minified config value>`, we could always write something to decrypt it :)